build.gradle.kts

```diff
@@ -69,11 +67,6 @@ allprojects {
                 }
             }
         }
-        mavenCentral {
```

Comment:
So we don't rely on mavenCentral at all anymore? Repox is sufficient?

Reply:
Yes. All the dependencies we rely on today are on Repox